code/modules/antagonists/changeling/powers/tiny_prick.dm

```diff
@@ -114,7 +114,7 @@
 		if(ismonkey(C))
 			C = C.humanize(TR_KEEPITEMS | TR_KEEPIMPLANTS | TR_KEEPORGANS | TR_KEEPDAMAGE | TR_KEEPVIRUS | TR_DEFAULTMSG | TR_KEEPAI)
 		var/datum/status_effect/ling_transformation/previous_transformation = C.has_status_effect(STATUS_EFFECT_LING_TRANSFORMATION)
-		C.apply_status_effect(STATUS_EFFECT_LING_TRANSFORMATION, new_dna, previous_transformation?.original_dna)
+		C.apply_status_effect(STATUS_EFFECT_LING_TRANSFORMATION, new_dna, (previous_transformation && istype(previous_transformation)) ? previous_transformation.original_dna : null)
```

Comment:
If I am reading the code right previous_transformation will never be null and is either the type or false. You don't need to  check for null here.